components/molecules/dropdown/components/dropdown.js

```diff
@@ -25,10 +25,10 @@ export const Dropdown = ({
   elementType,
   items,
   onChange,
-
+  isOpen,
```

Comment:
as this is not a controlled component, let's name this prop something like `defaultOpen` or so.